radio/src/targets/flysky/board.cpp

```diff
@@ -224,6 +224,7 @@ void boardInit()
   init2MhzTimer();
   init5msTimer();
   __enable_irq();
+  aux2SerialSetIdleCb(flysky_gimbal_loop);
```

Comment:
Please call flysky_gimbal_init here and register callback from it. If init implementation is causing trouble please comment it out but leave callback registration there.

Reply:
flysky_gimbal_init can not be called here as it does not allow the gimbal to communicate. I have comment it out and just left idlecallback here. Works well this way

Reply:
It should be called here to register serial callback (see original implementation), other part of implementation can be commented out with comment why.